packages/ember-truth-helpers/src/helpers/and.ts

```diff
@@ -9,11 +8,11 @@ export interface AndSignature {
   Return: boolean;
 }
 
-export default helper<AndSignature>((params) => {
+export default function and<T extends MaybeTruth[]>(...params: [...T]) {
```

Comment:
Since the `AndSignature` isn't being used anymore, should it be removed? Or is it left in for backwards compatibility with users who might be importing it?

(same goes for all the other signatures)

Reply:
Though it looks like some other helpers had their signatures deleted, so this one just might have been missed

Reply:
it was missed, thank you